nntrainer/include/databuffer.h

```diff
@@ -215,6 +248,7 @@ class DataBuffer {
   std::string val_file;
   std::string test_file;
   std::vector<std::string> labels;
+  bool validation[4];
```

Comment:
Looks like it seems to be `NBUFTYPE`
BTW, could I ask what `validation` is for?

Reply:
right. Thanks.